From ff308d76d72c33d9e0d2aa4e0301427406d25d4c Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Sun, 17 Apr 2016 15:34:51 +0300
Subject: [PATCH 23/38] ARM: i.MX6UL: dts: fix u-boot environment size

Fix u-boot environment partiton size in order to
comply with the u-boot environment settings.

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 arch/arm/boot/dts/imx6ul-cl-som-imx6ul.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6ul-cl-som-imx6ul.dtsi b/arch/arm/boot/dts/imx6ul-cl-som-imx6ul.dtsi
index 9fac193..9efaa76 100644
--- a/arch/arm/boot/dts/imx6ul-cl-som-imx6ul.dtsi
+++ b/arch/arm/boot/dts/imx6ul-cl-som-imx6ul.dtsi
@@ -395,7 +395,7 @@
 
 		partition@c0000 {
 			label = "uboot environment";
-			reg = <0xc0000 0x40000>;
+			reg = <0xc0000 0x2000>;
 		};
 
 		partition@100000 {
-- 
1.9.1

